diff --git a/BUILD.bazel b/BUILD.bazel
index 1234567..abcdefg 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -51,7 +51,6 @@
             "include/boost/spirit/home/x3/directive/expect.hpp",
             "include/boost/spirit/home/x3/directive/repeat.hpp",
             "include/boost/spirit/home/x3/nonterminal.hpp",
-            "include/boost/spirit/home/x3/nonterminal/detail/rule.hpp",
             "include/boost/spirit/home/x3/nonterminal/rule.hpp",
             "include/boost/spirit/home/x3/numeric.hpp",
             "include/boost/spirit/home/x3/numeric/bool.hpp",
